src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -158,7 +159,32 @@ class NavigationDrawer extends Component<CombinedNavigationProps> {
   }
 
   public render(): React.ReactElement {
-    const imgSrc = this.props.darkMode ? STFCLogoWhiteText : STFCLogoBlueText;
+    let navDrawerLogo = this.props.darkMode
+      ? STFCLogoWhiteText
+      : STFCLogoBlueText;
+
+    let altTxt = getString(this.props.res, 'alternative-text');
+
+    if (this.props.navigationDrawerLogo) {
+      if (this.props.darkMode) {
+        if (this.props.navigationDrawerLogo.dark) {
+          navDrawerLogo = this.props.navigationDrawerLogo.dark;
+          altTxt = this.props.navigationDrawerLogo.altTxt;
+        } else {
+          navDrawerLogo = STFCLogoWhiteText;
+          altTxt = getString(this.props.res, 'alternative-text');
```

Comment:
We don't need the else statements here right (or in the light mode section either)? As the default logo is already set on line 162